docs/src/main/paradox/java/http/routing-dsl/index.md

```diff
@@ -20,9 +20,29 @@ To use the high-level API you need to add a dependency to the `akka-http` module
 * [source-streaming-support](source-streaming-support.md)
 * [rejections](rejections.md)
 * [testkit](testkit.md)
+* [http-app](httpApp.md)
 
 @@@
 
+## Minimal Example
+
+This is a complete, very basic Akka HTTP application relying on the Routing DSL:
+
+@@snip [HttpServerMinimalExampleTest.java](../../../../../test/java/docs/http/javadsl/HttpServerMinimalExampleTest.java) { #minimal-routing-example }
+
+It starts an HTTP Server on localhost and replies to GET requests to `/hello` with a simple response.
+
+@@@ warning { title="API may change" }
```

Comment:
Please link to https://github.com/akka/akka/blob/master/akka-docs/rst/common/binary-compatibility-rules.rst#the-donotinherit-and-apimaychange-markers in the docs (thats akka docs, so direct link to /current/ will work well here IMO) so people can read up what this means.